docs/product/explore/profiling/transaction-vs-continuous-profiling.mdx

```diff
@@ -0,0 +1,77 @@
+
+We recommend that you call the `startProfiling` method right after the Sentry SDK is initialized so that you gain visibility at the earliest point in your application lifecycle. From then on, the profiler will keep collecting profiles and sending chunks to Sentry until `stopProfiling` is called.
+
+## Choosing Between Transaction and Continuous Profiling Mode
+
+Unfortunately, at this time it's not possible to use both profiling modes at the same time since they're mutually exclusive. Since the mode you intend to use is dependent on the SDK initialization arguments, profiling mode will have to be selected when the Sentry SDK is first initialized.
+
+To enable continuous profiling, you have to make sure that neither `profileSampleRate` nor `profilesSampler` are set as the values of the `Sentry.Init` call. If either of those values are set, the SDK will default to transaction-based profiling. When the SDK is configured for continuous profiling, the top level calls to start profiles will enable calls to the profiler. If the SDK is configured for transaction-based profiling, these calls will void and not trigger the profiler.
+
+Here's an example of how to enable continuous profiling in NodeJS:
+
+```javascript
+Sentry.Init({
+  dsn: "___PUBLIC_DSN___",
+  integrations: [nodeProfilingIntegration()]
+})
+
+Sentry.profiler.startProfiling();
+```
+
+If you want to keep using transaction-based profiling, then the options are the same. You can set either the `profilesSampleRate` or the `profilesSampler` option on the SDK.
+
+
+Here's an example of enabling transaction-based profiling in NodeJS:
+
+```javascript
+Sentry.Init({
+  dsn: "___PUBLIC_DSN___",
+  profileSampleRate: 0.1 // profiles 10% of transactions
+  integrations: [nodeProfilingIntegration()]
+})
+
+const transaction = Sentry.startTransaction();
+//code executed between these two calls is subject to profiling
+transaction.finish();
+```
+
+Note, that while the profiling mode can't be changed at runtime, it's fine for different projects or applications to use different profiling modes, or to switch modes.
+
+## Differences When Using Sentry
+
+The major difference between continuous profiling and transaction-based profiling when using Sentry, is that with continuous profiling you'll be able to visualize a flamegraph for your entire application. This means, that you'll be able to take a step back from the previous transaction-based view and look at your application's runtime as a whole, which makes it easier to prioritize the functions that are slowing down your entire application and not just one particular transaction.
+
+Otherwise, what you'll see in Sentry will be mostly the same. Entry points into profiling will continue to be supported by various parts of the performance product.
```

Comment:
> Entry points into profiling will continue to be supported by various parts of the performance product.

It's unclear to me what this means

Reply:
Yeah, lets remove this, folks shouldt have to care about it

Reply:
Removed this entire line at the end